README.md

```diff
@@ -169,16 +169,16 @@ The timestamps of the scan data published are computed to be close to reality an
 ### Transferred IOs
 
 The state of inputs and outputs of the PSENscan safety laser scanner are published as 
-([psen_scan_v2/IOState][]). The topic at `/\<name\>/io_states` is latched and always gives 
+[psen_scan_v2/IOState][]. The topic at `/<name>/io_states` is latched and always gives 
 the most recent pin state. On changes at the physical IOs, the changes are logged to
 the console and the new state is published to the topic.
 
-The [input pin state][psen_scan_v2/InputPins] contains information about
+The [input pin states][psen_scan_v2/InputPins] contains information about
```

Comment:
Please move 269ab4510e020874cd710603aa19626ac02d4c17 into a dedicated PR

Reply:
is now in #315